lib/skate_web/controllers/auth_controller.ex

```diff
@@ -14,16 +14,15 @@ defmodule SkateWeb.AuthController do
 
     current_time = System.system_time(:second)
 
-    User.upsert(username, email)
+    %{id: user_id} = User.upsert(username, email)
 
     conn
     |> Guardian.Plug.sign_in(
       AuthManager,
-      username,
+      "#{AuthManager.v2_resource_prefix()}#{user_id}",
       %{groups: credentials.other[:groups]},
       ttl: {expiration - current_time, :seconds}
     )
-    |> Plug.Conn.put_session(:username, username)
```

Comment:
this seems to have never been used (No references to `get_session(:username)`, and no tests failing on removal

Reply:
Sounds good - I can't think of anywhere it was used. Just watch out for any errors on dev I guess.